cloudslang-all/src/test/java/io/cloudslang/lang/api/SlangImplTest.java

```diff
@@ -56,6 +57,7 @@
  */
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = SlangImplTest.Config.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
```

Comment:
Do we really need `@DirtiesCOntext` on these tests? Will it reload the whole Spring context and thus impact performance? Can we just call menually the cleanup methods in a before/after method